src/test/java/com/ge/predix/uaa/token/lib/FastTokenServiceTest.java

```diff
@@ -1,303 +0,0 @@
-        fastTokenServices.afterPropertiesSet();
-        String accessToken = this.testTokenUtil.mockAccessToken(60, false);
-        OAuth2Authentication result = fastTokenServices.loadAuthentication(accessToken);
-        assertNotNull(result);
-
-        fastTokenServices.setRestTemplate(mockRestTemplate(true));
-        //Ensure the TokenKey for issuer times out
-        Thread.sleep(3100L);
-        accessToken = this.testTokenUtil.mockAccessToken(60, true);
-        result = fastTokenServices.loadAuthentication(accessToken);
-        assertNotNull(result);
-    }
-
-    @Test(expectedExceptions = InvalidSignatureException.class)
-    public void testLoadAuthenticationForUpdatedIssuerTokenSigningKeyNegative() throws Exception {
-        FastTokenServices fastTokenServices  = new FastTokenServices();
-        fastTokenServices.setTrustedIssuers(trustedIssuers());
-        fastTokenServices.setIssuerPublicKeyTTLMillis(3000L);
-        fastTokenServices.setRestTemplate(mockRestTemplate(false));
-        fastTokenServices.afterPropertiesSet();
-        String accessToken = this.testTokenUtil.mockAccessToken(60, false);
-        OAuth2Authentication result = fastTokenServices.loadAuthentication(accessToken);
-        assertNotNull(result);
-
-        fastTokenServices.setRestTemplate(mockRestTemplate(true));
-        //Ensure the TokenKey for issuer times out
-        Thread.sleep(3100);
-        //Try to reuse the token signed by the older token signing key
-        fastTokenServices.loadAuthentication(accessToken);
-    }
-
-    /**
-     * Tests that an token from the an untrusted issuer id throws an InvalidTokenException.
-     */
-    @Test(
-            expectedExceptions = InvalidTokenException.class,
-            expectedExceptionsMessageRegExp = ".*is not trusted because it is not in the configured list of trusted "
-                    + "issuers.")
-    public void testLoadAuthenticationWithUnstrustedIssuerId() throws Exception {
-        String accessToken = this.testTokenUtil.mockAccessToken("http://testzone1localhost:8080/uaa/oauth/token",
-                System.currentTimeMillis(), 60, false);
-        this.services.loadAuthentication(accessToken);
-    }
-
-    /**
-     * Tests that an expired token issues an InvalidTokenException.
-     */
-    @Test
-    public void testLoadAuthenticationWithExpiredToken() throws Exception {
-        String accessToken = this.testTokenUtil.mockAccessToken(System.currentTimeMillis() - 240000, 60);
-        try {
-            this.services.loadAuthentication(accessToken);
-            Assert.fail("Expected InvalidTokenException for expired token.");
-        } catch (InvalidTokenException e) {
-            System.out.println(e.getMessage());
-        }
-    }
-
-    /**
-     * Tests that an token that is valid for future use issues an InvalidTokenException.
-     */
-    @Test
-    public void testLoadAuthenticationWithFutureToken() throws Exception {
```

Comment:
we are not adding this test just because previous manual verification of token expiry logic removed?

Reply:
Added